### ISSUE TITLE
OpenFAST for multi-turbine platform simulations?

### ISSUE THREAD

**Issue by victormendoza1983** [OpenFAST/openfast#984]
Hello all, 
Is there anyway to setup simulations in OpenFAST for multi-turbine platforms?

![image](https://user-images.githubusercontent.com/97036019/151317595-bf0bdbda-581c-4d0a-b4cf-1587907ed946.png)

It would be nice to have this feature available!

**Comment by jjonkman** [comment]
Dear @victormendoza1983,

Without modifications to the source code, OpenFAST cannot model multiple wind turbines attached to a single floating substructure.

I know that Prof. Yoon Hyeok Bae of Jeju National University (the developer of the FEAMooring module) developed a version of FAST v8 that supported this functionality with two major limitations: (1) the floater is assumed to be rigid and (2) there is no aerodynamic or control interaction between the wind turbines.  But this version has not been incorporated into the public repository.  You may want to reach out to him to see what he can share.

Best regards,

**Comment by MikelIribas** [comment]
Yes this is possible and available some months ago. CENER has developed MUST (https://www.cener.com/en/wind-turbine-analysis-design/multi-wind-turbine-simulation-tool-must/).
MUST allows to simulate as much wind turbines as you want on a floater. There is a structural coupling, while it is also possible to have an aerodynamic coupling using AeroVIEW for wakes computation. This allows rotor-wake coupling, for a rotor with its own wake and also with neighbor’s wakes.  This is nowadays being used by companies developing multi-wind turbine floaters, and there are ongoing discussions with certification entities to speed up certification processes. Conctact me at miribas@cener.com, or any of our Wind Energy Department team for more details.

**Comment by jjonkman** [comment]
@victormendoza1983 -- I should have added that NREL has developed aerodynamic capability for modeling multi rotor systems in the AeroDyn driver of OpenFAST--see: https://github.com/OpenFAST/openfast/pull/688.  But NREL has not yet been funded to expand this work to include aero-elastic couplings.

@MikelIribas -- Can you clarify whether MUST is based on an extension of OpenFAST?  I seem to recall seeing a paper on MUST, but I now forget where I saw it.

Thanks,

**Comment by MikelIribas** [comment]
@jjonkman : MUST is an extension of OpenFAST for the multi-rotor concepts on a single floater. On the other hand, AeroVIEW (Aerodynamic Vortex fIlamEnt Wake) is an independent module, but compatible with OpenFAST.
You can see details on, for example:
“Validation of a free vortex filament wake module for the integrated simulation of multi-rotor wind turbines”. RenewableEnergy,179, pp. 1706–1718 https://www.sciencedirect.com/science/article/pii/S0960148121011605
and also in
“Lifting line free wake vortex fil-ament method for the evaluation of floating offshore wind turbines. First step: validation for fixed wind turbines”. In IWOTC, ASME 2019 2nd International Offshore WindTechnical Conference  https://asmedigitalcollection.asme.org/OMAE/proceedings-abstract/IOWTC2019/59353/V001T01A019/1071993

For more details, you should contact me or my colleagues Raquel Martin, rmartin@cener.com, or Jose Azcona  jazcona@cener.com.

**Comment by jjonkman** [comment]
Dear @MikelIribas,

Thanks for the links!  Indeed, the _Renewable Energy_ paper is one I recall seeing before.  From this paper, it looks like MUST is based on the work of Prof. Yoon Hyeok Bae for the structural part (with the same limitations), but with the addition of an alternative/improved aerodynamic model through your AeroVIEW code.  We should discuss at some point how to get some of your source code modifications into this OpenFAST repository.

Best regards,
